table/sst_file_writer.cc

```diff
@@ -170,7 +174,14 @@ SstFileWriter::SstFileWriter(const EnvOptions& env_options,
                              bool invalidate_page_cache,
                              Env::IOPriority io_priority, bool skip_filters)
     : rep_(new Rep(env_options, options, io_priority, user_comparator,
-                   column_family, invalidate_page_cache, skip_filters)) {
+                   column_family, invalidate_page_cache, skip_filters,
```

Comment:
So it is possible that the SstFileWriter instance is not changed during the period when db_seesion_id is created for several times due to DB reopen or other actions?